### PR TITLE
DOCSP-30594 Open Saved Pipeline

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
- Create task page for Open Saved Pipeline
  - Content taken from the original [Aggregation Pipeline Builder section](https://www.mongodb.com/docs/compass/current/aggregation-pipeline-builder/#open-a-saved-pipeline)

## STAGING
https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-30594-open-saved-pipeline/agg-pipeline-builder/open-saved-pipeline/

## JIRA
https://jira.mongodb.org/browse/DOCSP-30594

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6495a29e423952aeb9bc88b7

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/agg-pipeline-builder/open-saved-pipeline.txt [hunk 1]
```diff
@@ -12,4 +12,56 @@ Open a Saved Pipeline
...
+
+      In the top-left corner of the pipeline builder, click the folder icon to 
+      open the drop-down menu of saved pipelines.
+      
+      .. figure:: /images/compass/saved-pipeline-icon.png
+         :alt: Saved Pipeline Icon
+         
+   .. step:: Select an aggregation pipeline.
+      
+      In the drop-down menu, hover over the pipeline you want to open and
```
Comment (davemungo) [inline source/agg-pipeline-builder/open-saved-pipeline.txt:43]: As above To X, do Y - unless this is the usual Compass style

**Review by davemungo — COMMENTED** [review]
@ajhuh-mdb thanks for the updates. I'm not sure about the usual Compass wording style, a couple of the comments may be more appropriate for the server docs. If that's the case, just ignore them.

File: source/agg-pipeline-builder/open-saved-pipeline.txt [hunk 2]
```diff
@@ -12,4 +12,56 @@ Open a Saved Pipeline
    :depth: 1
    :class: singlecol
 
-TODO:
+You can use load saved aggregation pipelines to edit and re-use your pipelines
```
[the remark this answers is not in the document]
Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/open-saved-pipeline.txt:15]: Ah thanks for catching that — fixed.

File: source/agg-pipeline-builder/open-saved-pipeline.txt [hunk 3]
```diff
@@ -12,4 +12,56 @@ Open a Saved Pipeline
...
+         :alt: Saved Pipelines List 
+         
+   .. step:: Confirm your selection. 
+      
+      In the pop-up modal, click :guilabel:`Open Pipeline`.
+
+.. note::
+
+   Starting in |compass| 1.31, you can view your saved queries
+   and aggregation pipelines on the :guilabel:`My Queries` view
```
[the remark this answers is not in the document]
Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/open-saved-pipeline.txt:56]: Updated

File: source/agg-pipeline-builder/open-saved-pipeline.txt [hunk 4]
```diff
@@ -12,4 +12,56 @@ Open a Saved Pipeline
...
```
Comment (davemungo) [inline source/agg-pipeline-builder/open-saved-pipeline.txt:56]: Should this be 
in the My Queries view
or 
on the My Queries view [tab | screen | window]

**Review by davemungo — APPROVED** [review]
@ajhuh-mdb  thanks for the updates and info on the Compass style. One tiny question, otherwise this LGTM

File: source/agg-pipeline-builder/open-saved-pipeline.txt (hunk 4 above)
Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/open-saved-pipeline.txt:56]: will update to tab.

Comment (ajhuh-mdb) [reply source/agg-pipeline-builder/open-saved-pipeline.txt]: just confirmed with Julia that it wasn't an explicit product decision — will be changing these to "tab" across the relevant docs

**Pull request merged**